skype_send_message.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import bcrypt
+from skpy import Skype
+import time
+parser = argparse.ArgumentParser()
+parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--username', action="store")
+ShDo = "" # variable for complete work, ShDo = "shutdown"
+outputfile = "contact"
+print ("Welcome to 'Skype send message'. ")
+username = str(input("Type your username of Skype: "))
+password = str(input("Type your password of Skype: "))
```

Comment:
@idumenskyi lets ask for the password if it was not found in the hashed form saved previously

Reply:
@ekondrashev hashed.find(salt)?